proposals/stage-0/COLLAB_SUMMIT/README.md

```diff
@@ -0,0 +1,48 @@
+# OpenJS Collaborator Summit
+>  Stage 0
+
+## Champion
+
+Matteo Collina (@mcollina)
+
+## Description
+
+Organize an OpenJS Collaborator Summit twice a year, one in Europe and
+one in the US, following the lead of the [Node.js Collaborator Summit](https://github.com/nodejs/summit).
+
+The goal of the Collaborator Summit is to foster collaboration within
+and across projects in the Foundation.
+
+## Required Resources
+
+* Budget for venue
+* Budget for travel fund
+
+## Who would be responsible?
+
+A team/working group chartered under the CPC that includes
+representation from all interested projects and OpenJS staff.
+The team/working group will oversee the budget for the venue and the
+travel fund.
+
+## How would success be measured?
```

Comment:
Does it makes sense to add a `number of decisions made | problems resolved (if any)`?

Since this will be the venue where we will meet and discuss, this will be great ground to discuss and decide things. Definitely Not blocking on it.